src/packaging/common/packaging.properties

```diff
@@ -0,0 +1,11 @@
+# Common properties for building ZIP,GZ,RPM and DEB packages
+#
+# Properties defined here can be overridden with specific settings,
+# like in rpm/packaging.properties and deb/packaging.properties.
+
+# Default values for min/max heap memory allocated to elasticsearch java process
+packaging.elasticsearch.heap.min=256m
+packaging.elasticsearch.heap.max=1g
+
+# Simple marker to check that properties are correctly overridden
+packaging.type=tar.gz,gzip
```

Comment:
This marker is useful to check that a file has been filtered with the appropriate packaging.properties file. It's possible that I'll add a comment like this in some files:
```
# Built for ${project.name}-${project.version} (${packaging.type})
```
But you're right, for now it is not used. I can removed it.